### DISCUSSION TITLE
Is the tutorial not compatible with TCA 1.0.0 version?

### DISCUSSION THREAD

**Discussion by arusu0629** [pointfreeco/swift-composable-architecture#2341] [Q&A]
I'm getting a compile error in step 3 of [Your first presentation](https://pointfreeco.github.io/swift-composable-architecture/main/tutorials/composablearchitecture/02-01-yourfirstpresentation) Please check.

<img width="1292" alt="スクリーンショット 2023-08-02 15 50 59" src="https://github.com/pointfreeco/swift-composable-architecture/assets/5705360/ab362c95-1ce6-4693-a4ce-a520991c3577">

**Comment by mbrandonw** [comment]
It should be, but looks like we missed a small update in that code snippet. I just pushed a fix 9a8c3a046b, and so the docs should regenerate soon. Please let us know if you find any other problems.

**Reply by arusu0629** [reply]
@mbrandonw 
Thank you for your help!!!
I removed a compile error when I have done a fix https://github.com/pointfreeco/swift-composable-architecture/commit/9a8c3a046b327fd11bf7148a9f924eda7bec6464
